### DISCUSSION TITLE
Transactioncontext.jsx:129 An error occurred while sending transaction: Error: contract runner does not support sending transactions (operation="sendTransaction", code=UNSUPPORTED_OPERATION, version=6.12.1)

### DISCUSSION THREAD

**Discussion by mukilan2815** [adrianhajdin/project_web3.0#121] [Q&A]
Transactioncontext.jsx:129 An error occurred while sending transaction: Error: contract runner does not support sending transactions (operation="sendTransaction", code=UNSUPPORTED_OPERATION, version=6.12.1)

what is this error

 const sendTransaction = async () => {
    try {
      if (!ethereum) {
        alert("Please install MetaMask.");
        return;
      }

      const { addressTo, amount, keyword, message } = formData;
      const provider = new ethers.BrowserProvider(ethereum);

      const signer = provider.getSigner();

      const transactionsContract = new ethers.Contract(
        contractAddress,
        contractABI,
        signer
      );

      const parsedAmount = ethers.parseEther(amount);
      const Transactionhash = await transactionsContract.addToBlockchain(
        addressTo,
        parsedAmount,
        message,
        keyword
      );
      setIsLoading(true);
      console.log("Transactionhash", Transactionhash);
      await Transactionhash.wait();
      setIsLoading(false);
      console.log("Transactionhash sucess", Transactionhash);
      const transactionCount = await transactionsContract.getTransactionCount();
      setTransactionCount(transactionCount.toNumber());
    } catch (error) {
      console.error("An error occurred while sending transaction:", error);
      setError(
        "An error occurred while sending transaction. Please try again."
      );
    }
  };

**Comment by evgeny-s** [comment]
any update here?
